libs/dyn/convert/normalize.go

```diff
@@ -69,14 +69,18 @@ func typeMismatch(expected dyn.Kind, src dyn.Value) diag.Diagnostic {
 	}
 }
 
-func (n normalizeOptions) normalizeStruct(typ reflect.Type, src dyn.Value, seen []reflect.Type) (dyn.Value, diag.Diagnostics) {
+func (n normalizeOptions) normalizeStruct(typ reflect.Type, src dyn.Value, seen []reflect.Type, field_substitutions map[string]string) (dyn.Value, diag.Diagnostics) {
 	var diags diag.Diagnostics
 
 	switch src.Kind() {
 	case dyn.KindMap:
 		out := make(map[string]dyn.Value)
 		info := getStructInfo(typ)
 		for k, v := range src.MustMap() {
+			value, exists := field_substitutions[k]
```

Comment:
The CreateMonitor Struct uses a `full_name` but terraform takes in `table_name`. So  added support to substitute the keys of the field.

Reply:
Please take a look at `convert_job.go`.

Reply:
Thank You for pointing this out, removed it.